README.md

```diff
@@ -1,32 +1,133 @@
-# Home Office Forms Bootstrap [![Build Status](https://travis-ci.org/UKHomeOffice/hof-bootstrap.svg?branch=master)](https://travis-ci.org/UKHomeOffice/hof-bootstrap) [![npm version](https://badge.fury.io/js/hof-bootstrap.svg)](https://badge.fury.io/js/hof-bootstrap)
+# HOF (Home Office Forms) Bootstrap [![Build Status](https://travis-ci.org/UKHomeOfficeForms/HOF-bootstrap.svg?branch=master)](https://travis-ci.org/UKHomeOfficeForms/HOF-bootstrap) [![npm version](https://badge.fury.io/js/HOF-bootstrap.svg)](https://badge.fury.io/js/HOF-bootstrap)
+
+HOF-bootstrap is a wrapper function for HOF. It takes the hard work out of setting up your HOF service by encapsulating much of the boilerplate code in a single function. The bootstrap function takes many custom settings and options, but essentially maps your steps to Express routes and (optionally) starts the Express server.
+
+- [Getting started](#getting-started)
+  - [From HOF to HOF-Bootstrap](#from-hof-to-hof-bootstrap)
+  - [Starting from Scratch](#starting-from-scratch)
+    - [Default configuration](#default-configuration)
+      - [Typical project layout](#typical-project-layout)
+      - [Calling the bootstrap function](#calling-the-bootstrap-function)
+- [Interface](#interface)
+  - [start](#start)
+  - [stop](#stop)
+  - [use](#use)
+  - [server](#server)
+- [Custom configuration](#custom-configuration)
+  - [Options](#options)
+  - [Environment variables](#environment-variables)
+- [Routes](#routes)
+- [Resources](#resources)
+
+## Getting started
+
+`cd` into your project and install HOF-bootstrap.
+```
+$ npm install --save hof-bootstrap
+```
+
+### From HOF to HOF-bootstrap
+If you are familiar with HOF, the process of moving to HOF-bootstrap should be straightforward. The entry point of your app, will contain the boilerplate code that configures the apps' settings, routes, middleware, session storage and probably starts the server listening for connections.
+You can safely remove the boilerplate code and replace it with the bootstrap function. The following example invokes `bootstrap` with [routes](#routes). The custom configuration is used to tell HOF-bootstrap where to look for resources such as the views or controllers, or what session store to use.
+
+HOF-bootstrap includes all the same dependencies that HOF contains, so you can remove the HOF dependency from `package.json` and `npm uninstall --save hof`.
```

Comment:
I can specify the version of npm a la: https://docs.npmjs.com/files/package.json#engines